src/main/java/org/sonarlint/cli/Options.java

```diff
@@ -31,8 +31,10 @@
   private boolean version = false;
   private boolean showStack = false;
   private boolean interactive = false;
-  private String reportDir = null;
-  private String reportName = null;
+  private String htmlReport = null;
+  private String src = null;
+  private String tests = null;
+  private String charset = null;
   private String task;
 
   public static Options parse(String[] args) throws ParseException {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) The Cyclomatic Complexity of this method "parse" is 32 which is greater than 10 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AMethodCyclomaticComplexity)